### PR TITLE
Tests followup

### PR DESCRIPTION
**Pull request by isabelrios**

### CODE REVIEW

**Comment by isabelrios** [comment]
@JohanLorenzo just a very small PR to change the login (success and failure cases), should I add more changes here? Thanks!

File: test/selenium/lib/view/sign_up/view.js [hunk 1]
```diff
@@ -38,16 +38,26 @@ SetUpView.prototype = {
         return this.accessors.confirmPassword.sendKeys(text);
     },
 
-    tapSubmitButton: function() {
-        return this.accessors.submitButton.click();
-    },
+    successLogin: function(password, confirmPassword) {
+        this.accessors.insertPassword.sendKeys(password);
```
Comment (JohanLorenzo) [inline test/selenium/lib/view/sign_up/view.js]: Nit: `sendKeys` returns a Promise.

File: test/selenium/lib/view/sign_up/view.js [hunk 2]
```diff
@@ -38,16 +38,26 @@ SetUpView.prototype = {
...
-    tapSubmitButtonSignUp: function() {
         return this.accessors.submitButton.click().then(function(){
             return successfulPageView;
         });
     },
 
+    failureLogin: function(password, confirmPassword) {
+        return this.accessors.insertPassword.sendKeys(password)
+        .then(() => {
+        return this.accessors.confirmPassword.sendKeys(confirmPassword);
```
Comment (JohanLorenzo) [inline test/selenium/lib/view/sign_up/view.js]: Nit: Bad indentation

File: test/selenium/sessions_ui_test.js [hunk 3]
```diff
@@ -124,11 +112,8 @@ describe('sessions ui', function() {
...
 
       it('should accept matching, long-enough passwords', function() {
-          return setUpPage.typePassword('12345678').then(function() {
-              return setUpPage.confirmTypePassword('12345678');
-          }).then(function(){
-              return setUpPage.tapSubmitButtonSignUp();
-          }).then(function(successfulPageView) {
+          return setUpPage.successLogin(12345678, 12345678)
+          .then(function(successfulPageView) {
               return successfulPageView.successLogin();
```
Comment (JohanLorenzo) [inline test/selenium/sessions_ui_test.js]: Nit: `successLogin` sounds ambiguous now that we have `setUpPage.successLogin`. I'd change it to something closer to `loginMessage`

**Comment by JohanLorenzo** [comment]
r+. Looks good to me, modulo `successLogin` which should deal with promises (if we don't do it, we might have false positives).

No need to add tests in this PR, let's do it in a follow up. As a rule of thumb: the smaller the PR is, the easier it is (to review, modify and land).

**Pull request merged**